tasks/elasticsearch-parameters.yml

```diff
@@ -13,12 +13,12 @@
   when: es_config['http.port'] is not defined
 
 - name: debug message
-  debug: msg="WARNING - It is recommended you specify the parameter 'transport.tcp.port'"
-  when: es_config['transport.tcp.port'] is not defined
+  debug: msg="WARNING - It is recommended you specify the parameter 'transport.port'"
+  when: es_config['transport.port'] is not defined
 
 - name: debug message
-  debug: msg="WARNING - It is recommended you specify the parameter 'discovery.zen.ping.unicast.hosts'"
-  when: es_config['discovery.zen.ping.unicast.hosts'] is not defined
+  debug: msg="WARNING - It is recommended you specify the parameter 'discovery.seed_hosts'"
+  when: es_config['discovery.seed_hosts'] is not defined
```

Comment:
Same thing goes for this as I said for the `es_config['transport.port']` debug message.